manifests/prepare/package.pp

```diff
@@ -33,15 +33,17 @@
 
     if $::osfamily =~ /windows/ {
       $local_package_file_path = windows_native_path("${::puppet_agent::params::local_packages_dir}/${package_file_name}")
+      $mode = '0664'
```

Comment:
So, the original manifest was only dealing with the SLES/OSX cases where we needed to grab the package manually due to a lack of package management support; I think the fact that there was a POSIX mode was simply overlooked when it was reworked to include windows.  @Iristyle, @MikaelSmith I assume undef would be fine for the Windows case so long as the catalog is subsequently able to install the msi it has downloaded.